servers/quarkus-server/src/test/java/org/projectnessie/server/TestRest.java

```diff
@@ -516,6 +516,26 @@ void invalidTags(String invalidTagNameIn, String validHash) {
     );
   }
 
+  @Test
+  void testMerge() throws NessieConflictException, NessieNotFoundException {
```

Comment:
Can you move this test to `AbstractITVersionStore`?

Reply:
done